common/views/components/CollapsibleContent/index.tsx

```diff
@@ -80,8 +78,8 @@ const CollapsibleContent: FunctionComponent<Props> = ({
             setShowContent(!showContent);
           }}
         >
-          <Space as="span" h={{ size: 's', properties: ['margin-right'] }}>
-            <IconContainer open={showContent}>
+          <Space as="span" $h={{ size: 's', properties: ['margin-right'] }}>
+            <IconContainer>
```

Comment:
Like this prop was unused in the styled component itself - but maybe it's used in a different way?